src/SOFe/AwaitStd/AwaitStd.php

```diff
@@ -98,4 +100,23 @@ public function timeout(Generator $promise, int $ticks, $onTimeout = null) : Gen
 	public function awaitEvent(string $event, Closure $eventFilter, bool $consume, int $priority, bool $handleCancelled, object ...$disposables) : Generator {
 		return $this->eventAwaiter->await($event, $eventFilter, $consume, $priority, $handleCancelled, $disposables);
 	}
+
+	/**
+	 * Converts a Promise to a Generator by mapping onCompletion and
+	 * onError.
+	 *
+	 * @template V The promise's value
+	 * @throws PromiseRejectedException if the promise gets rejected
+	 * @param Promise<T> $promise
+	 * @return Generator<mixed, mixed, mixed, T>
+	 */
+	public function promiseToGenerator(Promise $promise): Generator {
```

Comment:
`AwaitStd::promiseToGenerator()` can be marked as static as it doesn't need or use `$this`.

Reply:
yes, please use static

Reply:
The function is now static